policies/recipes/steps.go

```diff
@@ -499,5 +500,14 @@ func executeCommand(cmd string, args []string, workDir string, runEnvs []string,
 			}
 		}
 	}
+	logger.Infof("Combined output for %q command:\n%s", cmd, o)
 	return err
 }
+
+func chown(file string, uid, gid int) error {
```

Comment:
is there some other way to chown on windows? I'm not sure it really makes much sense to change the owner anyway since I think it's just used for the tar command right now.